src/spells.cpp

```diff
@@ -850,7 +850,7 @@ bool InstantSpell::playerCastInstant(Player* player, std::string& param)
 			}
 
 			target = playerTarget;
-			if (!target || target->getHealth() <= 0) {
+			if (!target || target->isDead()) {
```

Comment:
```suggestion
			if (!target || target->isRemoved() || target->isDead()) {
```
Missing removed check?